ParisBNF/abb/BNFabb41.xml

```diff
@@ -341,7 +341,7 @@ xml:id="BNFabb41" type="mss">
                   <adminInfo>
                      <recordHist>
                         <source>
-                           <listBibl type="catalogue">
```

Comment:
you do not need to push records where there has been actually no changes

Reply:
you can undo the commit or overwrite with the version from master in this branch to have just one record pushed